lib/v2/elsevier/latest.js

```diff
@@ -0,0 +1,104 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+module.exports = async (ctx) => {
+    const resp = await got('https://www.sciencedirect.com');
+    const cookie = resp.headers['set-cookie'].join(' ');
+    const journal = ctx.params.journal;
+    const hostUrl = 'https://www.sciencedirect.com/journal/' + journal;
+
+    const hostResponse = await got({
+        method: 'get',
+        url: hostUrl,
+        headers: {
+            Host: 'www.sciencedirect.com',
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36',
+            Accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9',
+            Referer: 'https://id.elsevier.com/',
+            'Accept-Encoding': 'gzip, deflate, br',
+            'Accept-Language': 'zh-CN,zh;q=0.9',
+            cookie,
+        },
+    });
+
+    const $host = cheerio.load(hostResponse.data);
+    const jrnlName = $host('.anchor.js-title-link').text();
+    const ltstUrl = 'https://www.sciencedirect.com' + $host('.anchor.js-volume').attr('href');
+    let volume = '';
+    if (ltstUrl.match('suppl') !== null) {
+        volume = 'Volume ' + ltstUrl.match('vol/(.*)/suppl')[1];
+    } else if (ltstUrl.match('issue') !== null) {
+        volume = 'Volume ' + ltstUrl.match('vol/(.*)/issue')[1] + ' Issue ' + ltstUrl.match('/issue/(.*)')[1];
+    }
+
+    const ltstResponse = await got({
+        method: 'get',
+        url: ltstUrl,
+        headers: {
+            Host: 'www.sciencedirect.com',
```

Comment:
Why specified `www.sciencedirect.com` again in `Host` headers when `www.sciencedirect.com` is already specified in `ltstUrl`?

Reply:
Thanks your attention, I have modified it.
```javascript
const config = require('@/config').value;
const headers = {
    Accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9',
    'Accept-Encoding': 'gzip, deflate, br',
    'Accept-Language': 'zh-CN,zh;q=0.9',
    Referer: 'https://id.elsevier.com/',
    Host: host.replace('https://', ''),
    'User-Agent': config.ua,
    cookie,
};
```